### DISCUSSION TITLE
希望能开一个专门用于全国业主代表交流的纸飞机房间？ 方便大家同步信息还不被封。

### DISCUSSION THREAD

**Discussion by Leon0312** [WeNeedHome/SummaryOfLoanSuspension#644] [Ideas]
我觉得每个小区应该至少有那么一两个会用梯子的业主吧， 我觉得可以作为代表加入到纸飞机房间，大家商量对策，交流信息。现在微信群封的厉害。比如现在两百多各小区， 各有一位进来代表交流即可。

**Comment by masami1912** [comment]
支持，我也有这种想法，大家可以统一战线，应对措施

**Comment by JACWATA** [comment]
联系这些业主代表可能就会被盯上，支持，小心一点

**Comment by C-Joey** [comment]
风险比较大，操作难度大，建议谨慎

**Comment by MrChen2016-08-27** [comment]
不太可能，毕竟业主代表被收买的也有
